sal_interfaces/HVAC/HVAC_Events.xml

```diff
@@ -1,23 +1,92 @@
 <?xml version="1.0" encoding="utf-8"?>
 <?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsl"?>
 <SALEventSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsd">
-  <Enumeration>SummaryState_DisabledState,SummaryState_EnabledState,SummaryState_FaultState,SummaryState_OfflineState,SummaryState_StandbyState</Enumeration>
-  <Enumeration>DisabledState_DisabledState,DisabledState_EnabledState,DisabledState_FaultState,DisabledState_OfflineState,DisabledState_StandbyState</Enumeration>
-  <Enumeration>ChillerId_chiller01=1,ChillerId_chiller02=2,ChillerId_chiller03=3</Enumeration>
-  <Enumeration>CrackId_crack01=1,CrackId_crack02=2</Enumeration>
-  <Enumeration>FancoilId_fancoil01=1,FancoilId_fancoil02=2,FancoilId_fancoil03=3,FancoilId_fancoil04=4,FancoilId_fancoil05=5,FancoilId_fancoil06=6,FancoilId_fancoil07=7,FancoilId_fancoil08=8,FancoilId_fancoil09=9,FancoilId_fancoil10=10,FancoilId_fancoil11=11,FancoilId_fancoil12=12</Enumeration>
-  <Enumeration>ManejadoraId_manejadoraSblanca=1,ManejadoraId_manejadoraSlimpia=2</Enumeration>
-  <Enumeration>ManejadoraLowerId_manejadoraLower01=1,ManejadoraLowerId_manejadoraLower02=2,ManejadoraLowerId_manejadoraLower03=3,ManejadoraLowerId_manejadoraLower04=4</Enumeration>
-  <SALEvent>
-    <Subsystem>HVAC</Subsystem>
-    <EFDB_Topic>HVAC_logevent_detailedState</EFDB_Topic>
-    <Description>Event for the detailedState.</Description>
-    <item>
-      <EFDB_Name>detailedState</EFDB_Name>
-      <Description>Detailed state</Description>
-      <IDL_Type>long</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-  </SALEvent>
+  <Enumeration>
+    SummaryState_DisabledState,
+    SummaryState_EnabledState,
+    SummaryState_FaultState,
+    SummaryState_OfflineState,
+    SummaryState_StandbyState
+  </Enumeration>
```

Comment:
Please delete the SummaryState enum. It duplicates an enumeration that ts_sal already provides.

Reply:
Done